sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ConnectionStringProperties.java

```diff
@@ -25,25 +25,31 @@
     private static final String ENDPOINT = "Endpoint";
     private static final String SHARED_ACCESS_KEY_NAME = "SharedAccessKeyName";
     private static final String SHARED_ACCESS_KEY = "SharedAccessKey";
+    private static final String SHARED_ACCESS_SIGNATURE = "SharedAccessSignature";
+    private static final String SAS_VALUE_PREFIX = "sharedaccesssignature ";
     private static final String ENTITY_PATH = "EntityPath";
+    private static final String CS_WITH_ACCESS_KEY = "Endpoint={endpoint};SharedAccessKeyName={sharedAccessKeyName};"
```

Comment:
nit: change CS -> CONNECTION_STRING.  I avoid abbreviations unless I it's widely used.